LearnAndCodeAssignment_Week1/CheckArmstrongNumber.java

```diff
@@ -0,0 +1,34 @@
+package LearnAndCodeAssignment_Week1;
+
+import java.util.Scanner;
+
+public class CheckArmstrongNumber {
+	public static boolean checkIfArmstrongNumber(int numberToCheck) {
+        int sumOfDigitPowers = 0;
+        int digitCount = String.valueOf(numberToCheck).length();
+        int temporaryNumber = numberToCheck;
+
+        while (temporaryNumber > 0) {
+            int currentDigit = temporaryNumber % 10;
```

Comment:
why 10 is hardcoded?  Instead create a meaningful variable to give more clarity on why 10 has to be used here?